automation-tests/test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,264 @@
+    await prompt.confirm();
+
+    // Enter an org alias - yyyy-mm-dd-username-ticks
+    const currentDate = new Date();
+    const ticks = currentDate.getTime();
+    const day = ("0" + currentDate.getDate()).slice(-2);
+    const month = ("0" + (currentDate.getMonth() + 1)).slice(-2);
+    const year = currentDate.getFullYear();
+    const userName = utilities.currentUserName();
+    scratchOrgAliasName = `TempScratchOrg_${year}_${month}_${day}_${userName}_${ticks}_OrgAuth`;
+
+    await prompt.setText(scratchOrgAliasName);
+    await utilities.pause(1);
+
+    // Press Enter/Return.
+    await prompt.confirm();
+
+    // Enter the number of days.
+    await prompt.setText('1');
+    await utilities.pause(1);
+
+    // Press Enter/Return.
+    await prompt.confirm();
+
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Create a Default Scratch Org...', 5 * 60);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Create a Default Scratch Org... successfully ran');
+    if (successNotificationWasFound != true) {
+      const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Create a Default Scratch Org... failed to run');
+      if (failureNotificationWasFound == true) {
+        if (await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'organization has reached its daily scratch org signup limit', 5)) {
+          // This is a known issue...
+          utilities.log('Warning - creating the scratch org failed, but the failure was due to the daily signup limit');
+        } else if (await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'is enabled as a Dev Hub', 5)) {
+          // This is a known issue...
+          utilities.log('Warning - Make sure that the org is enabled as a Dev Hub.');
+          utilities.log('Warning - To enable it, open the org in your browser, navigate to the Dev Hub page in Setup, and click Enable.');
+          utilities.log('Warning - If you still see this error after enabling the Dev Hub feature, then re-authenticate to the org.');
+        } else {
+          // The failure notification is showing, but it's not due to maxing out the daily limit.  What to do...?
+          utilities.log('Warning - creating the scratch org failed... not sure why...');
+        }
+      } else {
+        utilities.log('Warning - creating the scratch org failed... neither the success notification or the failure notification was found.');
+      }
+    }
+    expect(successNotificationWasFound).toBe(true);
+
+    // Look for orgAliasName in the list of status bar items.
+    const statusBar = await workbench.getStatusBar();
+    const scratchOrgStatusBarItem = await utilities.getStatusBarItemWhichIncludes(statusBar, scratchOrgAliasName);
+    expect(scratchOrgStatusBarItem).not.toBeUndefined();
+  });
+
+  step('Run SFDX: Set a Default Org', async () => {
```

Comment:
Is this test equivalent to changing the default org in the org picker?

Reply:
Yes.  Also, what's in orgCreationAndAuthentication.e2e.ts is very similar to what's in scratchOrg.ts, only that orgCreationAndAuthentication.e2e.ts is more verbose and performs some additional checking.